### DISCUSSION TITLE
Question. Where is email templates responses ?

### DISCUSSION THREAD

**Discussion by Giorgos-Hellene** [YetiForceCompany/YetiForceCRM#14086] [General]
i can not find email templates. Could you help me how to find out ?
This was there on 5.3 version. 
![εικόνα](https://user-images.githubusercontent.com/50912403/105562387-a006bf00-5d22-11eb-8ebc-aca7eaea835f.png)

I think it is very important for every day use
thank you
![εικόνα](https://user-images.githubusercontent.com/50912403/105561324-0ee21900-5d1f-11eb-9733-9ee96fcc40b9.png)

**Comment by apcloic** [comment]
@Giorgos-Hellene 
Hi, I've just checked my YF system and templates responses are still present in YF 6.1
Maybe it's the same as email templates and require auto-login to be enabled ?
#13203

**Comment by Giorgos-Hellene** [comment]
@apcloic could you share a printscreen ??

i mean if i create an email template and choose type as message (not record). Where can we find it ?

**Reply by apcloic** [reply]
@Giorgos-Hellene 
See printscreen below for responses.
This is a compose email window when you click on any contact email.
![image](https://user-images.githubusercontent.com/2930560/105890523-c624ab80-600f-11eb-8ae7-706e9afd691a.png)

Regarding Email template, they are available when you go to Virtual Desk -> My mailbox.
Then click on Compose and then you will be able to select a template.

However, I've noticed that email template button is not working on compose email popup (when clicking directly on a contact email), you have to go the long way -> Virtual Desk -> My mailbox.

Regards

**Reply by Giorgos-Hellene** [reply]
You covered me !
i have just noticed that email template there is on gitstable but no on gitdeveloper.

Thank you @apcloic !!

**Comment by mariuszkrzaczkowski** [comment]
fixed https://github.com/YetiForceCompany/lib_roundcube/pull/66/files
check https://gitdeveloper.yetiforce.com/

**Reply by apcloic** [reply]
@mariuszkrzaczkowski 
Thanks a lot for the patch !
Note that in order to make it works, I also had to update /public_html/modules/OSSMail/roundcube/plugins/yetiforce/yetiforce.php
It works great when you go via Virtual Desk -> My mailbox ->compose new message.
However, it still doesn't work as expected, when you are on contact module, list view then click on a contact email address, roundcube compose popup opens but template button still does nothing.
I have check on https://gitdeveloper.yetiforce.com/
![yeti_email_template_not_working](https://user-images.githubusercontent.com/2930560/105985740-2c571000-609c-11eb-8757-1604cabcdaab.gif)

**Reply by mariuszkrzaczkowski** [reply]
we know it, we are thinking how to solve it, it is a complicated problem, therefore it has not been fixed yet

**Reply by Giorgos-Hellene** [reply]
> 
> 
> fixed https://github.com/YetiForceCompany/lib_roundcube/pull/66/files
> check https://gitdeveloper.yetiforce.com/

Thank you @mariuszkrzaczkowski
